examples/00-fluent/turbo_machinery.py

```diff
@@ -0,0 +1,815 @@
+        ],
+    }
+)
+
+solver_session.workflow.TaskObject["Define Turbo Topology"].AddChildAndUpdate()
+
+
+###############################################################################
+# Describe turbo surfaces
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# Define turbo-specific iso-surfaces.
+
+###############################################################################
+# .. image:: /_static/turbo_machinery_014.png
+#   :width: 500pt
+#   :align: center
+
+solver_session.workflow.TaskObject["Define Turbo Surfaces"].Arguments.set_state(
+    {
+        "IsoSurfaceNumList": ["surface 3", "surface 2", "surface 1"],
+        "NewIsoSurfaceNameList": ["twf_span_75", "twf_span_50", "twf_span_25"],
+        "NewIsoSurfaceValueList": ["0.75", "0.5", "0.25"],
+        "OldIsoSurfaceNameList": ["twf_span_3", "twf_span_2", "twf_span_1"],
+        "OldIsoSurfaceValueList": ["0.75", "0.5", "0.25"],
+    }
+)
+solver_session.workflow.TaskObject["Define Turbo Surfaces"].Execute()
+
+
+###############################################################################
+# Create report definitions and monitors
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Finalize the workflow and to create postprocessing objects such as contour plots
+# on the specified turbo-surfaces, as well as turbo-specific report definitions and monitors
+
+solver_session.workflow.TaskObject["Create Report Definitions & Monitors"].Execute()
+
+
+###############################################################################
+# Initialize flow field
+# ~~~~~~~~~~~~~~~~~~~~~
+# Initialize the flow field using hybrid initialization.
+
+solver_session.tui.solve.initialize.hyb_initialization()
+
+
+###############################################################################
+# Save case file
+# ~~~~~~~~~~~~~~
+# Save the case file as (``turbo_workflow.cas.h5``).
+
+solver_session.tui.file.write_case("turbo_workflow.cas.h5")
+
+###############################################################################
+# Solve for 100 iterations
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Solve for 100 iterations.
+
+# - Residuals
+
+###############################################################################
+# .. image:: /_static/turbo_machinery_015.png
+#   :width: 500pt
+#   :align: center
+
+# - Isentropic Efficiency of the Compressor
+
+###############################################################################
+# .. image:: /_static/turbo_machinery_016.png
+#   :width: 500pt
+#   :align: center
+
+# - Pressure Ratio
+
+###############################################################################
+# .. image:: /_static/turbo_machinery_017.png
+#   :width: 500pt
+#   :align: center
+
+#  - Total Mass Flow Rate at the Outlet
+
+###############################################################################
+# .. image:: /_static/turbo_machinery_018.png
+#   :width: 500pt
+#   :align: center
+
+solver_session.tui.solve.iterate(100)
```

Comment:
Could we work with less iterations for this example so that it runs quicker? As long as it doesn't make the results completely misleading. I think (may be wrong) this is also going to influence how long the documentation build takes. 

I would try to use as few iterations as possible that still gets somewhat decent results. Maybe 20 iterations?

Reply:
I think the tradeoff of having the iterations run ~4x faster is worth it (and improving doc build time). You could document that in these lines:
https://github.com/ansys/pyfluent/blob/1438197148adef64801dbbb1f936653a872bb725/examples/00-fluent/turbo_machinery.py#L767-L769

Saying that 100 iterations is recommended, but using 25 here as an example.